nf-test.config

```diff
@@ -0,0 +1,17 @@
+config {
+    // Location of nf-tests
+    testsDir "tests"
```

Comment:
```suggestion
    testsDir "."
```

This way all other tests (subwf and module) will also get picked up if you want to run them :)